src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000040> (melt pond)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Melt_pond") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000040> "A small water body, which forms via a melting processes occuring within a continuous mass of ice such a glacier or on sea ice.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000040> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000040> "melt pond"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000040> <http://purl.obolibrary.org/obo/ENVO_00000063>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000040> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_01000722>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000040> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002220> <http://purl.obolibrary.org/obo/ENVO_01000293>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000040> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_01000723>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000041> (brine channel)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://askabiologist.asu.edu/brine-channels") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000041> "A channel through which brine flows.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000041> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000041> "brine channel"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000041> <http://purl.obolibrary.org/obo/ENVO_00000395>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000041> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00003044>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000041> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000042>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000042> (brine channel formation process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary/term/frazil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000042> "A material accumulation process during which trapped brine droplets, between ice crystals formed during seawater freezing, form brine channels.")
```

Comment:
@pbuttigieg I thought a bit about it, brine channel formation process could either be a material transformation process or a material accumulation process, I think the latter is more useful. It's true that brine water results from the transformation of ice into meltwater with salt, but it's more useful to think of it as the brine water accumulating within places that would otherwise be a solid ice mass. I've added the following comment to make it more clear. 

> A brine channel formation process can be considered a material accumulation process because brine is accumulating due to a brine rejection process, a material transformation process during which brine is formed due to the expulsion of salt during seawater freezing process. This results in an accumulation of brine droplets, expelled salt dissolved in water, which remain in a liquid state due to their increased salinity.